t/op/try.t

```diff
@@ -281,4 +281,73 @@ no warnings 'experimental::try';
     is($caller, "main::B ($0 line $LINE)", 'try {} block is invisible to caller()');
 }
 
+# try/catch/finally
+{
+    my $x;
+    try {
+        $x .= "try";
+    }
+    catch ($e) {
+        $x .= "catch";
+    }
+    finally {
+        $x .= "finally";
+    }
+    is($x, "tryfinally", 'successful try/catch/finally runs try+finally but not catch');
+}
+
+{
+    my $x;
+    try {
+        $x .= "try";
+        die "Oopsie\n";
+    }
+    catch ($e) {
+        $x .= "catch";
+    }
+    finally {
+        $x .= "finally";
+    }
+    is($x, "trycatchfinally", 'try/catch/finally runs try+catch+finally on failure');
+}
+
+{
+    sub ff
+    {
+        try {
+            return "return inside try+finally";
+        }
+        catch ($e) {}
+        finally { "last value" }
```

Comment:
Should this test that the `finally` block actually gets invoked on `return` from the `try`?

Reply:
Good thinking. Added in b19ce2fd3d